src/model/censusdata/services/geodataApiDataService.js

```diff
@@ -0,0 +1,54 @@
+import { csvParse } from "d3-dsv";
+import { writeDataToMapObj } from "../../utils";
+
+const baseURL = "https://5laefo1cxd.execute-api.eu-central-1.amazonaws.com/dev/hello/skinny";
+
+export default class GeodataApiDataService {
+  async fetchAllDataForGeography(geographyId) {
+    const url = `${baseURL}?rows=${geographyId}`;
+    const response = await fetch(url);
+    const string = await response.text();
+    let data = new Map();
+    csvParse(string, (row, i, cols) => {
+      cols.forEach((col, i) => {
+        if (i == 0) {
+          data.set("geographyId", row[cols[0]]);
+        } else {
+          data.set(col, +row[col]);
+        }
+      });
+    });
+    return data;
+  }
+
+  async fetchSelectedDataForGeographyType(geoType, categories) {
+    const categoriesString = categories.toString();
+    let url = "";
+    if (geoType.toLowerCase().trim() == "lad") {
+      url = `${baseURL}?cols=geography_code,${categoriesString}&geotype=LAD`;
+    } else if (geoType.toLowerCase().trim() == "lsoa") {
+      url = `${baseURL}?cols=geography_code,${categoriesString}&geotype=LSOA`;
+    }
+    const response = await fetch(url);
+    const string = await response.text();
+    return writeDataToMapObj(string);
+  }
+
+  async fetchSelectedDataForGeographies(geoCodes, catCodes) {
+    const geoCodesString = geoCodes.toString();
+    const catCodesString = catCodes.toString();
+    const url = `${baseURL}?cols=geography_code,${catCodesString}&rows=${geoCodesString}`;
+    const response = await fetch(url);
+    const string = await response.text();
+    return writeDataToMapObj(string);
+  }
+
+  async fetchSelectedDataForBoundingBox(geoType, catCodes, bBox) {
```

Comment:
cosmetic, but `bbox` looks better than `bBox` IMO

Reply:
mmmm, I just feared it was a breach of strict camel casing 😬

Reply:
I guess `boundingBox` would satisfy both? Minor point though, feel free to ignore